ppx/src/reactjs_jsx_ppx.ml

```diff
@@ -500,7 +500,7 @@ let jsxMapper =
     Exp.apply
```

Comment:
the file rename matches `(library (name ...))` now, that's why I made that change.